server.js

```diff
@@ -1,29 +1,101 @@
-import express from 'express'
-import cors from 'cors'
-import mongoose from 'mongoose'
+import express from 'express';
+import cors from 'cors';
+import mongoose from 'mongoose';
 
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/happyThoughts"
-mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
-mongoose.Promise = Promise
+const mongoUrl =
+  process.env.MONGO_URL || 'mongodb://localhost/happyThoughtsAPI';
+mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true });
+mongoose.Promise = Promise;
 
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
+// Defines the port the app will run on. Default = 8080. Override example:
 //   PORT=9000 npm start
-const port = process.env.PORT || 8080
-const app = express()
+const port = process.env.PORT || 8080;
+const app = express();
 
-// Add middlewares to enable cors and json body parsing
-app.use(cors())
-app.use(express.json())
+// Middlewares to enable cors and json body parsing
+app.use(cors());
+app.use(express.json());
 
-// Start defining your routes here
+// Create a schema for the thoughts (input from user)
+const ThoughtSchema = new mongoose.Schema({
+  message: {
+    type: String,
+    minlength: 5,
+    maxlength: 140,
+    trim: true // Remove unnecessary blank spaces
+  },
+  name: {
+    type: String,
+    minlength: 3,
+    default: 'Anonymous'
+  },
+  hearts: {
+    type: Number,
+    default: 0
+  },
+  createdAt: {
+    type: Date,
+    default: () => Date.now()
+  }
+});
+
+// Pass the schema to the model
+const Thought = mongoose.model('Thought', ThoughtSchema);
+
+// Default route
 app.get('/', (req, res) => {
-  res.send('Hello world')
-})
+  res.send(
+    `This is Isabel's Happy Thoughts API. Add /thoughts to the URL bar to start`
+  );
+});
+
+// Sort thoughts by createdAt and return the most recent thoughts first (max. 20 of them)
```

Comment:
You did a great job with the comments throughout the code. They are descriptive and easy to understand.